cardano-cli/src/Cardano/CLI/EraBased/Options/Transaction.hs

```diff
@@ -384,3 +387,19 @@ pTransactionId =
     TransactionTxIdCmdArgs
       <$> pInputTxOrTxBodyFile
       <*> pTxIdOutputFormatJsonOrText
+
+pMetadataChecksOnProposals :: Parser MetadataCheck
+pMetadataChecksOnProposals =
+  Opt.flag EnableMetadataCheck DisableMetadataCheck $
+    mconcat
+      [ Opt.long "disable-metadata-checks-on-proposals"
```

Comment:
Right so now we have a common prefix: `disable-metadata-checks`

Repeating this is going to add to our already verbose commands. I'll have another look tomorrow to suggest an alternative.

Reply:
Tried to favor clarity over shortness with the flag names. The shortest I can come up with without loosing clarity is: 

```
--disable-vote-metadata-check
--disable-proposal-metadata-check
```